vtexdocs/devportal
# Feature/What's Next Cards

#### What is the purpose of this pull request?

To add the What's Next Card component and instances of it to the Quick Started (VTEX IO) page of the Developer's Portal.

Documented [here](https://www.notion.so/vtexhandbook/Criar-cards-da-p-gina-de-VTEX-IO-52db0485801a4f06bd0022cd3e2b8c23)

#### What problem is this solving?

The Quick Started (VTEX IO) page of the Developer's Portal should have cards leading to introductory documentation.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-54--elated-hoover-5c29bf.netlify.app/docs/vtex-io) and check the cards against [Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=6005%3A471890). Try different breakpoints!

#### Screenshots or example usage

#### Types of changes
